src/libratbag.c

```diff
@@ -611,6 +611,10 @@ ratbag_create_profile(struct ratbag_device *device,
 					   sizeof(*profile->resolution.modes));
 	profile->resolution.num_modes = num_resolutions;
 	profile->is_enabled = true;
+	profile->logo_led = zalloc(sizeof(*profile->logo_led));
+	profile->side_led = zalloc(sizeof(*profile->side_led));
+  profile->logo_led->profile = profile;
+  profile->side_led->profile = profile;
```

Comment:
sooner or later we'll need an allocator function for LEDs, might as well get started with it :) see e.g. ratbag_create_button, ratbag_create_profile, etc. for help